webpack.config.base.js

```diff
@@ -13,12 +13,41 @@ const typescriptLoaderRule = require('./webpack/typescript-loader-rule');
 // stripes-core (__dirname) before falling back to the platform or simply react
 const specificReact = generateStripesAlias('react');
 
+// A few details about stripes-config and entry
+//
+// Stripes config in Folio is currently assembled via webpack virtual module (https://github.com/sysgears/webpack-virtual-modules).
+// This plugin is a clever way to generate virtual (in memory) files dynamically.
+// What it means in practice is that it fools webpack to think that it deals with a real module/file
+// (like any other real module under node_modules) but in reality that file doesn't really exist.
+// Stripes config is "saved" under node_modules/stripes-config.js and it happens here:
+
+// https://github.com/folio-org/stripes-webpack/blob/master/webpack/stripes-config-plugin.js#L87
```

Comment:
Maybe replace with the [canonical URL](https://github.com/folio-org/stripes-webpack/blob/730ef33ee5a7799521408453b6e75653a21c3bfd/webpack/stripes-config-plugin.js#L87) so the reference stays valid even if that file changes?

Reply:
Great idea @zburke!

Reply:
[Github keyboard shortcuts](https://docs.github.com/en/get-started/using-github/keyboard-shortcuts#source-code-browsing). `y` transforms `blob/master/whatever.js` to `blob/$COMMIT_HASH/whatever.js`.

Reply:
Oh that's pretty handy! Thanks for sharing it.